sbi/inference/snre/snre_base.py

```diff
@@ -42,21 +42,14 @@ def __init__(
         show_round_summary: bool = False,
         logging_level: Union[int, str] = "warning",
     ):
-        r"""Sequential Ratio Estimation [1]
+        r"""Sequential Ratio Estimation [1].
 
         Args:
             classifier: Binary classifier.
             retrain_from_scratch_each_round: Whether to retrain the conditional
                 density estimator for the posterior from scratch each round.
-            summary_net: Optional network which may be used to produce feature
+            embedding_net: Optional network which may be used to produce feature
                 vectors f(x) for high-dimensional simulation outputs $x$.
-            classifier_loss: `sre` implements the algorithm suggested in Durkan et al.
```

Comment:
Thanks for the cleanup :) - this was intended for the children but never ended up there.